src/main/java/com/minecolonies/api/tileentities/storageblocks/ModStorageBlocks.java

```diff
@@ -0,0 +1,69 @@
+package com.minecolonies.api.tileentities.storageblocks;
+
+import com.minecolonies.api.MinecoloniesAPIProxy;
+import com.minecolonies.api.tileentities.AbstractTileEntityRack;
+import com.minecolonies.api.tileentities.storageblocks.registry.StorageBlockEntry;
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.core.tileentities.TileEntityRack;
+import com.minecolonies.core.tileentities.storageblocks.AbstractRackStorageBlockInterface;
+import com.minecolonies.core.tileentities.storageblocks.RackStorageBlockInterface;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.Optional;
+
+public final class ModStorageBlocks
+{
+    public static final DeferredRegister<StorageBlockEntry> DEFERRED_REGISTER = DeferredRegister.create(new ResourceLocation(Constants.MOD_ID, "storageblocks"), Constants.MOD_ID);
+
+    public static final RegistryObject<StorageBlockEntry> storageBlockRack;
+    public static final RegistryObject<StorageBlockEntry> storageBlockAbstractRack;
```

Comment:
An abstract rack isn't a thing, it's a root class for racks.
And to be fair, this root class is frankly highly redundant.

Reply:
Done.